src/queries/user/recommendation/newest.ts

```diff
@@ -44,10 +44,10 @@ export const newest: RecommendationToNewestResolver = async (
     .as('newest')
 
   const [_countRecord, articles] = await Promise.all([
-    MAX_ITEM_COUNT, // knex.select().from(query.clone().limit(MAX_ITEM_COUNT)).count().first(),
+    MAX_ITEM_COUNT, // always 500 // knex.select().from(query.clone().limit(MAX_ITEM_COUNT)).count().first(),
     knex
       .select()
-      .from(query.clone().limit(MAX_ITEM_COUNT))
```

Comment:
Remember why we used `clone' here?

Reply:
the common pattern of

```js
const [ articlesCount, articles ] = Promise.all([
  500, // query.count(),
  query.clone().orderBy(...)...
])
```

no longer need the `.clone()` when the other clause is turning to const 500;